src/appliance-onboarding-script/appliance_setup/avs/avsarconboarder/retriever/arcaddon/arc_add_on_retriever.py

```diff
@@ -0,0 +1,32 @@
+import logging
+
+from ...entity.CustomerResource import CustomerResource
+from ...entity._credentials import Credentials
+from ...exception import DataNotFoundException
+from ...executor.azcli._AzCliExecutor import AzCliExecutor
+from ...constants import Constant
+from ...entity.AzCli import AzCli
+from ...retriever._retriever import Retriever
+
+
+class ArcAddOnRetriever(Retriever):
+    instance = None
+    arc_add_on_url = Constant.MGMT_URL + "addons/arc" + "?" + Constant.API_VERSION + "=" + Constant.PREVIEW_API_VERSION_VALUE
```

Comment:
The below constants are not properly named anymore:
STABLE_API_VERSION_VALUE = "2021-06-01" -> This version doesn't support Arc AddOn
PREVIEW_API_VERSION_VALUE = "2022-05-01" -> This version is not in preview anymore.

Reply:
let's take this up in a new PR.